2-0-samples.md

```diff
@@ -390,29 +396,68 @@ items". The Sample Alias will have been filled in with aliases that end in D\_1
 
 <a name="qcs"/>
 
-# 4. Adding Sample QCs
+# 4. Sample QCs
 
-#TODO
+Several QC methods are supported in MISO. Currently they must be entered individually per
+sample, but there are plans to add bulk QC in the near future.
+
+## 4.1 Adding Sample QCs
+
+Add a QC for one of your existing aliquots.
+
+1. On the _Samples_ page, enter your project name in the search box.
+1. Click on one of your gDNA aliquots. Its alias should end with `_D_1`.
+1. Near the bottom of the page, beside the QCs heading, click _Options_ and select
+_Add Sample QC_.
+1. For _QC Date_, select today's date
+1. _Method_: QuBit
+1. _Results_: 50
+1. Click _Add_. The QC will be added to the table
 
 <a name="boxes"/>
 
 # 5. Working with Boxes
 
-#TODO customize for samples
-
 {% include boxes.md %}
 
 
 <a name="trouble"/>
 
 # Troubleshooting
 
-#TODO 
+**How do you work with multiple samples derived from the same identity (i.e. DNA and RNA from one tissue slide, DNA from metastasis and control)**
+
+On the _Samples_ page, you can enter the identity's alias into the search box to filter the list.
+If the samples you are looking for have more of their alias in common, you can enter as much as
+they share to further narrow the search. e.g. to find all stock samples made from tissue
+`PROJ_0001_Ly_R_nn_1-1`, you could search for "PROJ_0001_Ly_R_nn_1-1_D_S".
+
+Once you have found all of the samples you wish to work with, select them by clicking the
+checkboxes on the left, and choose a bulk action from the menu at the bottom.
+
+**How do I correct a sample that has the wrong information, e.g. wrong tissue type?**
+
+Most sample attributes may be modified either individually, or in bulk. To access the
+_Edit Sample_ page for a single sample, click the sample's name or alias on the _Samples_ page.
+To bulk edit a set of samples, click the checkboxes next to them on the _Samples_ page, choose
+_Update Selected_ from the _Bulk actions_ menu, and click _Go_.
+
+For attributes which cannot be changed via MISO (e.g. Sample Class), please email gsi@oicr.on.ca
+or file a JIRA ticket in GSI Common to get assistance from the MISO team.
+
+**What if I need to change the parent of a sample? (e.g. for swaps)**
+
+Please email gsi@oicr.on.ca or file a JIRA ticket in GSI Common to get assistance from the MISO team.
+
+**What do I have to enter if receiving a pre-made library?**
+
+MISO does not yet accept direct receipt of libraries. In cases where libraries are
+received, you are required to first create the aliquot and then propagate the library from it.
+The aliquot may be created by using the Create Sample page (single or bulk), or by propagating
+from an existing tissue or stock Sample. Propagating libraries is detailed in a later tutorial.
```

Comment:
Please link to libraries tutorial, preferably directly to the session for propagating libraries.